docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -125,21 +125,25 @@ Specify one or more columns of the table and must be surrounded by parentheses.
 - `NULLS FIRST` - Specifies that nulls sort before non-nulls. This is the default when DESC is specified.
 - `NULLS LAST` - Specifies that nulls sort after non-nulls. This is the default when DESC is not specified.
 
+### NULLS NOT DISTINCT
+
+When creating an unique index, by default, null values are not considered equal, allowing multiple nulls in the column. The NULLS NOT DISTINCT option modifies this and causes the index to treat nulls as equal. With this option, you can enforce that only one NULL value is permitted, aligning with use cases where NULL should represent an absence of value rather than a unique entity.
```

Comment:
With NULL NOT DISTINCT options, the nulls are treated as equivalent (and not equal). In Postgres, NULL = NULL is not true. So "causes the index to treat nulls as equal" is slightly misleading. 

PG doc for the feature says (https://www.postgresql.org/about/featurematrix/detail/392/):
"By default, NULL values are treated as distinct entries. Specifying NULLS NOT DISTINCT on unique indexes / constraints will cause NULL to be treated as not distinct, or in other words, equivalently."